pytest_fixtures/core/contenthosts.py

```diff
@@ -143,3 +144,17 @@ def container_contenthost(rhel7_contenthost, default_sat):
         rhel7_contenthost.execute(f'yum -y install {service}')
         rhel7_contenthost.execute(f'systemctl start {service}')
     return rhel7_contenthost
+
+
+@pytest.fixture(scope='module')
+def set_remote_execution_by_default(request):
```

Comment:
don't forget to inherit the module_target_sat fixture now that you're using it
```suggestion
def set_remote_execution_by_default(request, module_target_sat):
```

Reply:
doh

Reply:
Fixed now, thank you